Makefile.toml

```diff
@@ -485,6 +561,16 @@ done
 # modules for a given release.
 LINK_REPO_TARGETS=("--link-target ${BUILDSYS_KMOD_KIT_PATH}")
 
+# Ensure that we link an OVA if the current variant is vmware*
+if [[ "${BUILDSYS_VARIANT}" =~ ^vmware.* ]]; then
```

Comment:
We could change this to test `[ -s "${BUILDSYS_OVF_TEMPLATE}" ]` instead, since that's what controls OVA generation now.

Reply:
Ah good catch. Fixing...